tests/fixtures/noqa/noqa.py

```diff
@@ -1,4 +1,5 @@
-# -*- coding: utf-8 -*-  # noqa: WPS323
+#!/usr/bin/env pytho # noqa: WPS453 WPS323
```

Comment:
I am sure that we should not be allowed to add comments to shebangs.
Let's report this violation on line `0`, so we would not be able to ignore it with a comment.

Reply:
Do you mean that we should add a new violation message for when a comment is appended to a shebang ? In this case can I just remove this test from `noqa.py` ?

Reply:
No, I don't mean that. 🙂 

I mean that we should raise this violation on `0` line.

Reply:
Do we want to keep the shebang test in `noqa.py` then ? Because I can't see how we can test that `# noqa` does not work for a specific violation.

Reply:
Yes, we can remove that.